app/models.py

```diff
@@ -696,7 +696,7 @@ def remove_domain(self, name):
             db.session.delete(sd)
             db.session.flush()
 
-    def update_domain_assessment_status(self, name_or_id, status, user=''):
+    def update_domain_assessment_status(self, name_or_id, status, user='', data={}):
```

Comment:
I think it will be clearer if you called this audit_data
```suggestion
    def update_domain_assessment_status(self, name_or_id, status, user='', audit_data={}):
```